draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2792,29 +2793,34 @@ repeats the connectivity check that produced this valid pair (by enqueueing
 the pair that generated the check into the TRIGGERED CHECK QUEUE), this time with
 the USE-CANDIDATE attribute. The connectivity check should succeed (since the
 previous did), causing the nominated flag value of that and only that pair
-to be set to true.
-</t>
-<t>Eventually, there will be only a single nominated pair in the VALID LIST for
-each component. Once the state of the CHECK LIST is set to Completed, that exact
-pair is selected by ICE for sending and receiving media for that component.
+to be set to true. However, if the connectivity check fails <xref target="sec-client-failure"/>, 
+the controlling agent MUST remove the candidate pair from the VALID LIST, set 
+the candidate pair state to Failed and set the CHECK LIST state to Failed.</t>
+<t>Eventually, if the nominations succeed, there will be only a single nominated 
+pair in the VALID LIST for each component. Once the state of the CHECK LIST is 
+set to Completed, that exact pair is selected by ICE for sending and receiving 
+media for that component.
 </t>
 <t>The criterion details for stopping the connectivity checks and for
 selecting a pair for nomiation, are outside the scope of this specification.
 They are a matter of local optimization. The only requirement is that the
 agent MUST eventually pick one and only one candidate pair and generate a
 check for that pair with the USE-CANDIDATE attribute present.
 </t>
-<t>The controlled agent SHOULD select the candidate pair nominated by the
-controlling agent if the controlled agent is receiving Binding responses
-associated with that candidate pair. Before the agent has received Binding
-responses associated with the candidate pair, the agent can send media on
-any candidate for which it has received Binding responses.
-</t>
-<t>If more than one candidate pair is nominated by the controlling agent, the
-controlled agent SHOULD select the candidate pair with the highest priority.
-</t>
-<t>
-NOTE: A controlling agent that does not support this specification (i.e. it
+<t>If the controlled agent accepts the nomination request from the controlling
+agent, the controlled agent MUST select the nominated candidate pair, if the 
+controlled agent is receiving Binding responses associated with that candidate 
+pair. Before the agent has received Binding responses associated with the 
+candidate pair, the agent can send media on any candidate for which it has 
```

Comment:
I don't understand what the last phrase "if the controlled agent is receiving Binding responses associated with that candidate pair" means.  Is that just meaning we can't select until the pair is in the valid list?  If so, why not just say that (you can't select something not in the valid list)?